src/tests.c

```diff
@@ -4324,7 +4324,7 @@ void test_ecdsa_sign_verify(void) {
     secp256k1_scalar one;
     secp256k1_scalar msg, key;
     secp256k1_scalar sigr, sigs;
-    int recid;
+    int recid = 0;
```

Comment:
Can we set this init to `-1` instead to make sure that the `CHECK` below will fail if `recid` is never written to?